documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1394,116 +1413,113 @@ The setting of the PostgreSQL connector configuration property, `decimal.handlin
 When the `decimal.handling.mode` property is set to `precise`, the connector uses the Kafka Connect `org.apache.kafka.connect.data.Decimal` logical type for all `DECIMAL` and `NUMERIC` columns. This is the default mode.
 
 .Mappings when `decimal.handling.mode` is `precise`
-[cols="15%a,15%a,35%a,35%a",options="header"]
+[cols="28%a,17%a,55%a",options="header"]
 |===
 |PostgreSQL data type
 |Literal type (schema type)
-|Semantic type (schema name)
-|Notes
+|Semantic type (schema name) and Notes
```

Comment:
I got to admit I'm not super-excited about this merge from a logcial perspective; but I see how it can help with arranging the columns more flexibly, so I'm not opposing :)

Reply:
I understand. But I can't think of another alternative that would work downstream. For example, it's possible to format a table to merge rows or columns and present notes that way, but merging rows or columns hinders screen readers for visually impaired folks, and so downstream, we are not allowed to format tables that way.